znnl/models/flax_model.py

```diff
@@ -100,6 +101,9 @@ def __init__(
                 The default value is trace_axes(-1,), which reduces the NTK to a tensor
                 of rank 2.
                 For a full NTK set trace_axes=().
+        store_on_device : bool, default True
```

Comment:
This is a cool feature, is it new in NT?

Reply:
I actually don't know. have seen it for the first time too and put it in.